doc/build-instructions/Build_Instructions_V8.md

```diff
@@ -378,12 +377,12 @@ set INCLUDE=C:\Program Files\Debugging Tools for Windows (x64)\sdk\inc;%INCLUDE%
 set LIB=C:\Program Files\Debugging Tools for Windows (x64)\sdk\lib;%LIB%
 ```
 
-Not all of the shared libraries that are included with Visual Studio 2010 are registered during installation.
-In particular, the `msdia100.dll` libraries must be registered manually.
+Not all of the shared libraries that are included with Visual Studio 2013 are registered during installation.
+In particular, the `msdia120.dll` libraries must be registered manually.
 To do so, execute the following from a command prompt:
```

Comment:
As per https://github.com/eclipse/openj9/issues/11477#issuecomment-748214560 the following needs to be done as administrator, which can be added here, and for 11, and 15 as well.

Reply:
@jdekonin if you can update this, I think the PR will be ready for merge.

Reply:
Thank you for pointing that out @pshipton.  PR updated.

Reply:
I don't think `Administrator` should be capitalized. Can you please add this bit to the jdk11 and 15 file as well.

Reply:
Seems both Administrator and administrator (like `Run as administrator`) are used.